localgov_guides.links.menu.yml

```diff
@@ -0,0 +1,6 @@
+localgov_guides.settings:
+  title: Localgov Guides Settings
+  description: Configure the localgov_guides module
+  parent: system.admin_config_system
```

Comment:
I think Content authoring would be a better parent? Would be interested in others opinions.

Reply:
Me too. I couldn't figure if there was a standard localgovdrupal approach for these sorts of settings.